my-app/src/Components/CardPost/CardPostBig/CardPostBig.tsx

```diff
@@ -0,0 +1,37 @@
+import { useNavigate } from "react-router-dom";
+import style from './CardPostBig.module.scss';
```

Comment:
style is object and if you have a lot classes which are used in your component you can destructure it
as example
const { wrapper, wrap, wraptextcontent  } = styles;